README.md

```diff
@@ -98,3 +98,31 @@ hvac_zones = orm.session.query(Location)\
                         .all()
 print(f"Model has {len(hvac_zones)} HVAC Zones")
 ```
+## Validate with Shape Constraint Language
+
+The module utilizes the [pySHACL](https://github.com/RDFLib/pySHACL) package to validate a building ontology
+against the Brick Schema, its default constraints (shapes) and user provided shapes.
+
+```python
+from brickschema.validate import Validate, ResultsSerialize
+from rdflib import Graph
+import sys
+
+dataG = Graph()
+dataG.parse('myBuilding.ttl', format='turtle')
+v = Validate()
```

Comment:
I'll comment later, but this should be called `Validator` rather than `Valiate`, and the serialization of the results should be wrapped around a `Result` object:


```python
v = Validator()
v.addShapeFile('extraShapes.ttl')
result = v.validate(dataG)
if not result.conforms:
    print(result.output) # the output is a string; it can be printed manually)
    result.graph # gets the result graph
```

Reply:
I like the name Validator.  Currently the validate() function faithfully returns what's returned by pyshacl.validate().  I think it's easier to explain that to the user.  If the user wants further info (extra info), they should call violaitonList (or get it as a property) which has the extra info attached.

Reply:
pyShacl is stateless, but our implementation is not. People are unfamiliar enough with pyshacl that I don't think it's worth it to adopt their stateless API on top of our stateful one. It's better to be consistent, and to me that means sticking with the stateful approach. This is *especially* confusing when you are splitting up the result of the validation across properties of the `Validator` class as well as what is returned by the call to `validate`

Reply:
I don't mind abandoning the pyshacl appearance and return whatever we like to return.

Reply:
I'm laying down the new API here before proceeding:

1. Abandon the attempt to keep the validate() function consistent with pyshacl's.  The validate function will take a list of ontology graphs and a list of shape graph.  The results.graph returns the results_graph from pyshacl.validate while result.output returns OUR output text (with extra info).
2. With item 1, the addShapeGraph and addShapeFile methods become unnecessary.  So does the class for result serialization.